main.go

```diff
@@ -21,6 +25,8 @@ func failf(msg string, args ...interface{}) {
 }
 
 func main() {
+	const TestName = "Flutter test results"
```

Comment:
Is there a nned for uppercase variable names?

Reply:
I will check the naming conventions for constants

Reply:
https://stackoverflow.com/questions/22688906/go-naming-conventions-for-const

"The standard library uses camel-case" shouldn't we use camel-case as well?

Reply:
I believe it needs to be uppercase for exported constants only.

Reply:
There is example here for mixed case: http://golang.org/doc/effective_go.html#mixed-caps

Reply:
done